pkg/app/master/command/debug/handle_kubernetes_runtime.go

```diff
@@ -96,6 +96,23 @@ func HandleKubernetesRuntime(
 		return
 	}
 
+	if commandParams.ActionListDebuggableContainers {
+		xc.Out.State("action.list_debuggable_containers",
+			ovars{"namespace": nsName})
+
+		result, err := listK8sDebuggableContainers(ctx, api, nsName, "")
+		if err != nil {
+			logger.WithError(err).Error("listK8sDebuggableContainers")
+			xc.FailOn(err)
+		}
+
+		for cname, iname := range result {
+			xc.Out.Info("debuggable.container", ovars{"name": cname, "image": iname})
```

Comment:
@eharris128 if the intent is to show all debuggable containers across all pods then we need the pod name for each record we are printing here.

Reply:
Agreed. Definitely a necessary extension.